modules/logging-operator-upgrading-all-ns.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+:_content-type: PROCEDURE
+[id="logging-operator-upgrading-all-ns_{context}"]
+= Upgrading the Cluster Logging Operator to watch all namespaces
+
+In logging 5.7 and older versions, the Cluster Logging Operator only watches the `openshift-logging` namespace.
+If you want the Cluster Logging Operator to watch all namespaces on your cluster, you must redeploy the Operator. You can complete the following procedure to redeploy the Operator without deleting your logging components.
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+* You have administrator permissions.
+
+.Procedure
+
+. Delete the subscription by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-logging delete subscription <subscription>
+----
+
+. Delete the Operator group by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-logging delete operatorgroup <operator_group_name>
+----
+
+. Delete the cluster service version (CSV) by running the following command:
++
+[source,terminal]
+----
+$ oc delete clusterserviceversion cluster-logging.<version>
+----
+
+. Redeploy the Cluster Logging Operator by following the "Installing Logging" documentation.
```

Comment:
I guess a link isn't necessary here since if you CTRL+F it pops up, but maybe it would be useful?

Reply:
I'd love to have a link here, but it's a module, so I can't have an xref 😄